### ISSUE TITLE
Run tests with spaces in the extension installation path

### ISSUE THREAD

**Issue by DonJayamanne** [microsoft/vscode-jupyter#157]
In light of the issues identified with tests/code failing with spaces in the path, we might as well run our tests with spaces in the paths.

**By tests i mean** 
* Functional tests with Python
* Unit tests (do we need to?. No harm, hence why not)

**We could run as follows (in the flaky or insider Workflow):**
* Functional tests (with python) on latest python & linux
* Functional tests (with python) on latest python & windows
* Unit tests on linux & windows

Basically the idea is to minimize the number of jobs running.

**Comment by rchiodo** [comment]
Probably never going to do this
